receiver/prometheusreceiver/config_test.go

```diff
@@ -278,41 +278,3 @@ func TestKubernetesSDConfigWithoutKeyFile(t *testing.T) {
 	gotErrMsg := err.Error()
 	require.Equal(t, wantErrMsg, gotErrMsg)
 }
-
-func TestFileSDConfigJsonNilTargetGroup(t *testing.T) {
```

Comment:
can you add a testcase that validates that no error happens in case the file does not exist?

Reply:
thanks. Updated.